src/posit/connect/jobs.py

```diff
@@ -0,0 +1,292 @@
+from typing import Literal, Optional, TypedDict, overload
+
+from typing_extensions import NotRequired, Required, Unpack
+
+from .resources import Active, ActiveFinderMethods, ActiveSequence, Resource
+
+JobTag = Literal[
```

Comment:
a nit beyond all nits: wondering if this list should be organized a bit more cleanly - we could alphabetize within job types or something like that. 

if Connect adds new job types and starts returning them to the SDK, will that cause issues?

Reply:
It wouldn't break anything. The type system would be outdated, but we expect this to happen over time. Since we provide wildcard fallbacks for public methods (e.g., find_by, create, etc.), the user's program shouldn't be affected.